CONTRIBUTING.md

```diff
@@ -23,6 +23,7 @@ Repository | Component | Description | Devfile | Documentation
 [che-theia](https://github.com/eclipse-che/che-theia) | | Theia IDE integrated in Che. | [devfile](https://github.com/eclipse-che/che-theia/blob/main/devfiles/che-theia-all.devfile.yaml) | [doc](https://github.com/eclipse-che/che-theia/blob/main/CONTRIBUTING.md)
 ---| [generator](https://github.com/eclipse-che/che-theia/tree/main/generator) | `che:theia init` CLI to prepare and build che-theia | | |
 [chectl](https://github.com/che-incubator/chectl) | | The CLI to install Che, create and start workspaces and devfiles | [devfile](https://github.com/che-incubator/chectl/blob/master/devfile.yaml) | [doc](https://github.com/che-incubator/chectl/blob/master/CONTRIBUTING.md)
+[che-code](https://github.com/che-incubator/che-code) | | Fork of code-oss to work with Eclipse Che | [devfile](https://github.com/che-incubator/che-code/blob/master/devfile.yaml) | [doc](https://github.com/che-incubator/che-code/blob/master/README.md)
```

Comment:
> `code-oss`

Officially it's called `Code - OSS`. I think it matters here.

Reply:
updated again